### ISSUE TITLE
Change enabled badge color to improve accessibility

### ISSUE THREAD

**Issue by rladdusaw** [voormedia/flipflop#48]
On the feature page the enabled badge's background color is #5cb85c has insufficient contrast with the foreground color (2.48:1) and fails WCAG 2.0 levels AA (4.5:1) and AAA (7:1) contrast standards.

The background color should be changed to increase contrast:
A value of #238637 (4.63) would meet AA and #036600 (7.23:1) would meet AA and AAA.

**Comment by rladdusaw** [comment]
I'm happy to create a PR to implement this change, just let me know the exact color you would like to use!

**Comment by rolftimmermans** [comment]
Hi, thanks for your report. `#238637` looks okay to me. `#036600` seems too dark, making the green harder to distinguish. Please feel free to make a pull request.
